apidef/oas/default_test.go

```diff
@@ -1662,7 +1672,7 @@ func TestOAS_importAuthentication(t *testing.T) {
 		assert.Nil(t, authentication)
 	})
 
-	t.Run("add first authentication in case of OR condition", func(t *testing.T) {
+	t.Run("import all authentication methods for OR condition", func(t *testing.T) {
```

Comment:
seems that the test case was changed - I would rather add a complete new test and keep the old one instead, so that we can make sure that we keep the confidence that the backwards-compatible behvaior is not broken